<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
border-image: url(:/image/login_back.jpg)
}
*{
font: 12pt &quot;等线&quot;;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="login_btn">
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="username_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>用户名:</string>
      </property>
      <property name="buddy">
       <cstring>username_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="password_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
      <property name="buddy">
       <cstring>passowrd_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passowrd_edit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85); 
    border-radius: 3px; 
    background: white;   
    selection-background-color: green; 
}
 
QLineEdit:hover {
    border: 1px solid blue; 
}</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="username_edit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85); 
    border-radius: 3px; 
    background: white;   
    selection-background-color: green; 
}
 
QLineEdit:hover {
    border: 1px solid blue; 
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="wrong_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>用户名或密码错误！</string>
   </property>
  </widget>
  <widget class="QLabel" name="hello_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;欢迎来到北邮教务系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_edit</tabstop>
  <tabstop>passowrd_edit</tabstop>
  <tabstop>login_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
